### PR TITLE
Fec 8696 contrib presets

### PR DESCRIPTION
**Pull request by esakal**

### CODE REVIEW

File: src/common/ui-wrapper.js [hunk 1]
```diff
@@ -10,6 +10,7 @@ class UIWrapper {
 
   constructor(player: KalturaPlayer, options: KPOptionsObject) {
     const config: UIOptionsObject = options.ui;
+    (config.presetComponents || (config.presetComponents = [])).push(...player._localPlayer.presetComponents);
```
Comment (esakal) [inline src/common/ui-wrapper.js]: @OrenMe the code above assume that `presetComponents` can be provided in two ways:
1. from each plugin using the new interface
2. statically from the page that configured the ui

do you want to keep both or only the first one?

Comment (yairans) [reply src/common/ui-wrapper.js]: And if we want to keep both, what's about merging them?

Comment (yairans) [inline src/common/ui-wrapper.js]:
```suggestion
    (config.presetComponents || (config.presetComponents = [])).concat(player._localPlayer.presetComponents);
```

Comment (OrenMe) [reply src/common/ui-wrapper.js]: Keep both, what happens if app provides same component as plugin, let's assume different use cases, like app wanting to override default plugin component.
A rule of thumb in player, app(or user) always gets precedence over default config.

Comment (esakal) [reply src/common/ui-wrapper.js]: done

**Comment by esakal** [comment]
not relevant anymore

**Pull request closed without merging**